rebasehelper/cli.py

```diff
@@ -151,6 +152,13 @@ def build_parser():
             help="continue previously interrupted rebase"
         )
         parser.add_argument(
+            "--colors",
```

Comment:
This should be rather `color`, to stay consistent with other tools using argument like this.